### PR TITLE
Allow matthiasnoback/sf-di-test ^1.0

### PR DESCRIPTION
**Pull request by greg0ire**
I am targetting this branch, because this is BC.

Refs https://github.com/sonata-project/dev-kit/issues/216

This is pedantic because it's only about a development dependency.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "symfony/security": "^2.2 || ^3.0",
         "symfony/security-acl": "^2.2 || ^3.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6",
+        "matthiasnoback/symfony-dependency-injection-test": "^1.0",
```
Comment (OskarStark) [inline composer.json]: shouldn't we allow both?

Comment (greg0ire) [inline composer.json]: It's `require-dev`, so it's better to drop the unstable versions IMO.

**Comment by OskarStark** [comment]
Please check Travis

File: composer.json (hunk 1 above)
Comment (greg0ire) [inline composer.json]: Well let's allow both, since sf-di-test 1 require sf/yaml 2.7

**Pull request merged**